src/probnum/diffeq/_odesolver.py

```diff
@@ -79,11 +165,32 @@ def postprocess(self, odesol):
         """Process the ODESolution object before returning."""
         return odesol
 
-    def method_callback(self, time, current_guess, current_error):
+    def method_callback(self, state):
         """Optional callback.
 
         Can be overwritten. Do this as soon as it is clear that the
         current guess is accepted, but before storing it. No return. For
         example: tune hyperparameters (sigma).
         """
         pass
```

Comment:
currently, this is still part of the ODE solver. I think it is superfluous, but since it is used for the wrapped scipy solvers, I think it cannot be removed safely.